src/tsc/tsc.ts

```diff
@@ -21,4 +21,8 @@ if (ts.sys.setBlocking) {
     ts.sys.setBlocking();
 }
 
+if (!ts.sys.args.find(arg => /^--?skipJSDocParsing$/.test(arg))) {
+    ts.sys.args.push("--skipJSDocParsing");
+}
+

```

Comment:
can you move this inside execute command line so our verifyTsc etc can run with this just like tsc.